django_admin_geomap/__init__.py

```diff
@@ -89,11 +89,19 @@ def set_common(self, request, context):
         return context
 
     def changelist_view(self, request, extra_context=None):
+        # Obtain original response from Django
+        response = super().changelist_view(request, extra_context=extra_context)
+
+        # Obtain final queryset from ChangeList object to supported filtered queries
+        change_list = response.context_data['cl']
```

Comment:
The intermediate variable change_list can probably be removed and assigned immediately to change_list_queryset